QuasarHelios2015/src/org/team1540/quasarhelios/Clamp.java

```diff
@@ -15,18 +18,20 @@
 public class Clamp {
 	public final BooleanOutput openControl = BooleanMixing.combine(Igneous.makeSolenoid(0), Igneous.makeSolenoid(1));
 	public final FloatOutput heightControl;
-	
+
 	public final FloatInputPoll heightReadout;
 
 	public Clamp() {
-		FloatStatus desiredHeight = new FloatStatus();
-		heightControl = desiredHeight;
+		heightControl = new FloatStatus();
+
+		FloatFilter limit = FloatMixing.limit(0.0f, 1.0f);
+		FloatInput desiredHeight = limit.wrap((FloatInput) heightControl);
```

Comment:
Casting like this is usually bad form. It means that the compiler can't do type-checking for you.